Users/Tutorial.rst

```diff
@@ -264,24 +267,32 @@ That looks horrible, doesn't it? Let's fix it!
     $ coala -S python.bears=PEP8Bear python.files=\*\*/\*.py \
     --apply-patches --save
     # other output ...
+    Executing section cli...
     Executing section python...
     [INFO][11:03:37] Applied 'ApplyPatchAction' for 'PEP8Bear'.
     [INFO][11:03:37] Applied 'ApplyPatchAction' for 'PEP8Bear'.
 
 coala would now fix all spacing issues and without bothering you again.
 
+.. note::
+
+    When you try the above example, you may get a warning, saying that all
+    settings in the ``cli`` section are implicitly inherited to all
+    other sections (if they do not override their values). It also advises
+    us to change the name of that section to avoid unexpected behavior.
```

Comment:
I think this note shall somehow go into the paragraph before, saying "There will be warnings about the `cli` section bla bla.....". We should advise the people earlier that `cli` should be renamed!

Reply:
See comment below, I should have read further :3

Reply:
You just changed perspective mid paragraph. First you started with "you may get a warning...", then you suddenly switched to "..advises us to change..". Please stick to one perspective.

Reply:
Here's the output :--
```
[WARNING][22:26:33] 'cli' is an internally reserved section name. It may have been generated into your coafile while running coala with `--save`. The settings in that section will inherit implicitly to all sections as defaults just like CLI args do. Please change the name of that section in your coafile to avoid any unexpected behavior.
```
Notice the part--  *__Please__ change the name of that section in your coafile to avoid any unexpected behavior.*

Don't you think that this request to change is an advice inside the warning?